packages/app/src/files-manager/helpers/display-space.test.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { displaySpace } from "App/files-manager/helpers/display-space"
+
+const MBvalue = 124
+const GBvalue = 15000
```

Comment:
this is still MBvalue 😉

Reply:
Names changed in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)

Reply:
Names changed in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)